src/kagglehub/auth.py

```diff
@@ -14,3 +17,19 @@ def login():
     set_kaggle_credentials(username=username, api_key=api_key)
 
     logger.info("Kaggle credentials set.")
+
+    if validate_credentials is False:
+        return
+
+    try:
+        api_client = KaggleApiV1Client()
+        api_client.get("/hello")
+        logger.info("Kaggle credentials successfully validated.")
+    except requests.exceptions.HTTPError as e:
+        invalid_credentials_error = 403
```

Comment:
You can move this to a "constant" at the top of this file like: https://github.com/Kaggle/kagglehub/blob/95612cb2a41139a005fec5926e27c55802469b2e/src/kagglehub/cache.py#L8